packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/vsphere-users.controller.js

```diff
@@ -59,8 +62,17 @@ export default class DedicatedCloudVsphereUsersCtrl {
     return this.DedicatedCloud.getUserDetails(this.productId, params).then(
       ({ data: users, meta }) => {
         const userFormat = users.map((user) => {
-          set(user, 'loginUsername', user.login.split('@')[0]);
-          set(user, 'loginDomain', user.login.split('@')[1]);
+          const [userName, domain] = user.login.split('@');
+          set(user, 'loginUsername', userName);
+          set(user, 'loginDomain', domain);
+          let userType = user.type || 'user';
+          if (
+            has(user, 'identityProviderType') &&
+            user.identityProviderType === 'iam'
+          ) {
+            userType = 'role';
```

Comment:
**nitpick** Can be maintained in the const file

Reply:
Since, 'enabled' string is used in multiple places, you can define it as a constant and reuse it.

Reply:
Ok, done